### PR TITLE
Various edits

### PR DESCRIPTION
**Pull request by blazman**
Aiming to do a top-to-bottom review with readability edits and contributions as I go. This is P1-18, as well as some commits addressing #402, removed confidentiality agreement activity (covered by assessment plan), and other minor fixes.

### CODE REVIEW

**Comment by blazman** [comment]
Confirmed this does build with Documentation Builder

File: en/document_matter/audit_lifecycle.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 * **Vulnerabilities** Vulnerabilities are specific flaws or attributes of an asset susceptible to attack.
-* **Threats** A threat is a specific, possible attack or occurrence that could harm the organization.  If a bucket of oily rags is a vulnerability, a fire is the threat - and mitigations would be rules against leaving oily rags around as well as fire extinguishers, smoke detectors, remote backup policies, and evacuation planning.
+* **Threats** A threat is a specific, possible attack or occurrence that could harm the organization.  If a bucket of oily rags is a vulnerability, a fire is the threat - and capacity would be rules against leaving oily rags around as well as fire extinguishers, smoke detectors, remote backup policies, and evacuation planning.
 
-To make SAFETAG approachable, a core evaluation template which links together a series of specific objectives, each with a variety of linked activities, that contribute towards the goals and their required information needs is represented here.  Experienced Auditors will likely come up with their own approaches, and the SAFETAG project welcomes such contributions.
```
Comment (blazman) [inline en/document_matter/audit_lifecycle.md]: Not sure what 'core evaluation template' this is referring to. Doesn't make much sense so have deleted it. I recall there was a 'metro map', which is not currently being displayed - is this what was meant?

**Comment by joncamfield** [comment]
Re: Style guide -- Should we change it from American English with its love for the Z? If the S (here and generally) are more globally standardi...sed upon, that's just one massive regex to fix it all and make it less America-centric.

**Comment by blazman** [comment]
I don't really have a strong opinion. When I looked into it a bit I saw that American spelling is also common in large english as a second language areas like south america and parts of Asia, whereas british is used throughout the commonwealth. I just thought it should be specified somewhere.

**Comment by blazman** [comment]
BTW I may have gotten confused around the reference files and index files named *.overview.md so 
375a097 may need to be tweaked. The way I understood it was that *.overview.md was the index file being used when building the shorter overview document, but when I wanted to update some text in the main guide I found that the active text was in an .overview.md file, so I tried to swap it to the regular version of the file.

**Comment by joncamfield** [comment]
Thanks!

**Pull request merged**